.github/workflows/consuming.yml

```diff
@@ -102,8 +102,29 @@ jobs:
       - name: Make sure ${{ matrix.project }} is using the built Shipyard image
         run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' ${{ matrix.project }}/Dockerfile.dapper
 
-      - name: Run all linters
-        run: make -C ${{ matrix.project }} lint
+      - name: Run gitlint
+        if: always()
+        run: make -C ${{ matrix.project }} gitlint
+
+      - name: Run golangci-lint
+        if: ${{ always() && matrix.project != 'submariner-charts' }}
+        run: make -C ${{ matrix.project }} golangci-lint
+
+      - name: Run markdownlint
+        if: always()
+        run: make -C ${{ matrix.project }} markdownlint
+
+      - name: Run packagedoc-lint
+        if: ${{ always() && matrix.project != 'cloud-prepare' && matrix.project != 'coastguard' && matrix.project != 'lighthouse' && matrix.project != 'submariner-charts' && matrix.project != 'submariner-operator' }}
```

Comment:
I gave up trying to wrap this, if you can get it working/wrapped I'd love to use that.

Reply:
```
[22:18:18.] [linting]$ make yamllint
yamllint --strict .
./.github/workflows/consuming.yml
  118:141   error    line too long (216 > 140 characters)  (line-length)
  122:141   error    line too long (281 > 140 characters)  (line-length)
```

Seems like the linter is going to insist, keep us honest.